features/release_calendar_response.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Proxy returns response from Release Calendar
+
+    The proxy can be called with Release Calendar URLs (for instance, "/releases/adoption"),
+    when this happens we want to ensure that the proxy forwards the URL to the Release Calendar. 
+
+  Scenario Outline: Proxy returns response from Release Calendar
+    Given Release calendar will send the following response with status "200":
+      """
+      Mock response from Release Calendar
+      """
+    When the Proxy receives a GET request for "/releases/some-path"
+    Then the HTTP status code should be "200"
```

Comment:
Shouldn't you be using `I should receive the same, unmodified response from Release Calendar`? You have created that step, but you're not really using it anywhere.